packages/react/src/components/Breadcrumb/index.test.tsx

```diff
@@ -1,60 +1,55 @@
-import React, { createRef } from 'react';
-import { shallow } from 'enzyme';
-import {
-  default as Breadcrumb,
-  BreadcrumbLink,
-  BreadcrumbItem
-} from 'src/components/Breadcrumb';
-import axe from '../../../axe';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Breadcrumb, { BreadcrumbLink, BreadcrumbItem } from './index';
+import axe from '../../axe';
 
 test('should render breadcrumbs', () => {
-  const breadcrumb = shallow(
+  const { container } = render(
```

Comment:
When we've been updating our tests, we've been trying to follow better [test writing practices](https://testing-library.com/docs/guiding-principles) with the following guideline:

> It should be generally useful for testing the application components in the way the user would use it.

In this instance, we want this to be looking for a navigation type element, as that's what the user would see. To meet that expectation, we would write (as an example):

```jsx
render(<Breadcrumb aria-label="breadcrumb">...</BreadCrumb>
await expect(screen.getByRole('navigation', { name: 'breadcrumb' })).toBeInTheDocument()
await expect(screen.queryByRole('listitem')).toHaveLength(3)
```

As far as classes and other props, we can split those out into other tests specific to those properties as seen as an example below:

https://github.com/dequelabs/cauldron/blob/5d92f1e2b49f6a5ce814de4c996c285c79d4fab4/packages/react/src/components/Link/Link.test.tsx#L41-L59

Reply:
Hi @scurker, I did update the code to adhere to RTL best practices. Thank you for the feedback!